server/routes/APImanager.js

```diff
@@ -0,0 +1,43 @@
+const urllib = require('urllib')
+class APImanager {
+    constructor() {
+        this.data = []
+        this.teamToIDs = {
```

Comment:
when you code you always want one source of truth. You have data in `this.teamToIDs` and `this.idsToTeams` that is exactly the same. Instead of switching keys around, do it with methods in your functions.